codalab/apps/api/serializers.py

```diff
@@ -54,10 +54,29 @@ class Meta:
 
 class PhaseSerial(serializers.ModelSerializer):
     start_date = serializers.DateField(format='%Y-%m-%d')
+    is_active = serializers.Field()
```

Comment:
is this part necessary? I don't believe you need this here?

Reply:
The reason I do this is because `is_active` is a property on the `CompetitionPhase` model. In the DRF docs [here](https://www.django-rest-framework.org/api-guide/serializers/#specifying-fields-explicitly).

<img width="850" alt="Screen Shot 2019-11-18 at 9 44 38 AM" src="https://user-images.githubusercontent.com/21694127/69076398-38e7cf00-09e8-11ea-9b81-53d590ccf014.png">

Reply:
for future reference: this broke without `is_active = serializers.Field()`